.travis.yml

```diff
@@ -5,6 +5,7 @@ language: go
 
 go:
   - 1.7
```

Comment:
`1.7` means `1.7.0` in Travis - this should actually be `1.7.x` and below `1.8.x`